python_modules/dagster/dagster/_core/definitions/automation_condition_sensor_definition.py

```diff
@@ -83,6 +83,12 @@ class AutomationConditionSensorDefinition(SensorDefinition):
         description (Optional[str]): A human-readable description of the sensor.
         allow_backfills (bool): If set to True, will emit a backfill on any tick where more than one partition
             of any single asset is requested, rather than individual runs. Defaults to True.
+        use_user_code_server (bool): (experimental) If set to True, this sensor will be evaluated in the user
```

Comment:
I would also note in the docblock that this ensures that production code remains in sync with any testing code, eliminating version skew.